ansible/roles/airflow/tasks/af-container.yml

```diff
@@ -0,0 +1,36 @@
+---
+- file: path=/srv/{{ dockname }} state=directory mode=0750 owner={{ login }} group={{ login }}
+- stat: path=/srv/{{ dockname }}/airflow.cfg
+  register: afcfg
+- command: cp -a /srv/etc/af-share-{{ airflow_tag }}/airflow.cfg /srv/{{ dockname }}/airflow.cfg
+  when: not afcfg.stat.exists
+- name: airflow service -- {{ command }}
+  docker_container:
+    image: openobservatory/airflow{{ image_ext | default('') }}:{{ airflow_tag }}
+    name: "{{ dockname }}"
+    hostname: "{{ dockname }}"
+    networks: [ "{{ {'name': 'af'} | combine(
+      ipv4_address | default(False) | ternary({'ipv4_address': ipv4_address}, {})
+    ) }}" ]
+    purge_networks: true
+    volumes: "{{ [
+      '/srv/etc/af-share-' + airflow_tag + '/passwd:/etc/passwd:ro',
+      '/srv/etc/af-share-' + airflow_tag + '/group:/etc/group:ro',
+      '/srv/' + dockname + ':/usr/local/airflow:rw',
+      '/srv/airflow-dags:/usr/local/airflow/dags:ro',
+    ] | union(
+      volume_ext | default([])
+    ) }}"
+    env:
+      LOAD_EX: n
+      EXECUTOR: Celery
+      FERNET_KEY: "{{ airflow_fernet_key }}"
+      REDIS_HOST: af-redis
+      REDIS_PASSWORD: "{{ airflow_redis_password }}"
+      POSTGRES_HOST: af-psql
+      POSTGRES_USER: airflow
+      POSTGRES_PASSWORD: "{{ airflow_postgres_password }}"
+    command: "{{ command }}"
+    user: "{{ passwd[login].id }}:{{ passwd[login].id }}"
+    restart_policy: on-failure
```

Comment:
Note that if we use the `unless-stopped` restart_policy and for some reason we manually stop a container this container will not be brought up upon reboot (since it was in a stopped condition).
The cases that we will need to stop a production container will be very few and for sure we can use other ways (if it's needed) to stop containers and not let allow to auto starting them again.